index.js

```diff
@@ -47,6 +51,10 @@ console.log('hello world')
  */
 
 // TODO: ADD CODE HERE
+const price = 9.99;
+const quantity = 10;
+const total = price * quantity;
+console.log(Math.round(total));
```

Comment:
Small note here, the `total` is being rounded to the nearest whole number instead of two decimal places (should be `$99.90` instead of `100`).